src/llvm/di.rs

```diff
@@ -80,7 +80,19 @@ impl DIFix {
 
                         // we detect this is a variadic enum if the child element is a DW_TAG_variant_part
                         let elements = LLVMGetOperand(value, 4);
```

Comment:
this call is pointless if `metadata` is a forward declaration, right?

Reply:
Yes it is pointless :). 
Removing this requires changing a bit the logic but I can do it